server/tests/users.js

```diff
@@ -0,0 +1,101 @@
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import server from '../index';
+
+chai.use(chaiHttp);
+
+const should = chai.should(); // eslint-disable-line
+const user = {
+    userName: 'deschant',
+    password: 'password',
+    email: 'deschant@epic.com',
+};
+const userLog = {
+    userName: 'deschant',
+    password: 'password',
+};
+
+describe('signup', () => {// eslint-disable-line
+    it('should return an object', (done) => { // eslint-disable-line
+        chai.request(server)// eslint-disable-line
+            .post('/api/v1/auth/signup')
+            .send(user)
+            .end((err, res) => {
+                res.body.should.be.an('Object');
+                done();
+            });
+    });
+    it('should have property data', (done) => { // eslint-disable-line
+        chai.request(server)// eslint-disable-line
+            .post('/api/v1/auth/signup')
+            .send(user)
+            .end((err, res) => {  // eslint-disable-line           
+                res.body.should.have.property('data')
+                    .which.is.an('array');
+                done();
+            });
+    });
+    it('should have property status', (done) => { // eslint-disable-line
+        chai.request(server)// eslint-disable-line
+            .post('/api/v1/auth/signup')
+            .send(user)
+            .end((err, res) => {  // eslint-disable-line           
+                res.body.should.have.property('status');
```

Comment:
Wouldn't be better to assert the exact status instead of just asserting it's a property in the body?

Reply:
Oh okay. Let me try to do so.